sections/exchange/hooks/useExchange.ts

```diff
@@ -602,6 +601,30 @@ const useExchange = ({
 		}
 	);
 
+	useEffect(() => {
+		if (exchangeTxn.hash) {
+			monitorTransaction({
```

Comment:
Could be nice to extract this to a separate function to reduce duplicated code, it can then be called from both here and from the other submit tx.

Reply:
Yeah. Saw that, but was too lazy to change it. Will fix now.

Reply:
I was thinking we could do the whole thing couldn't we, like a function called `monitorSubmitTx(hash)` as there's still some unnecessary duplication?